test/docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
       - mongodb
 
   agent:
-    container_name: kontena-agent
+    container_name: e2e-kontena-agent
```

Comment:
oh, I've managed to put this same change to other PR also... :/ I'll revert this